Libplanet.Net/Transports/NetMQTransport.cs

```diff
@@ -335,7 +335,6 @@ public void Dispose()
 
                 if (_router is { } router && !router.IsDisposed)
                 {
-                    _router.Unbind($"tcp://*:{_listenPort}");
                     _router.Dispose();
```

Comment:
```suggestion
                    // We omitted _router.Unbind() with intention due to hangs.
                    // See also: https://github.com/planetarium/libplanet/pull/2311
                    _router.Dispose();
```

it may help future readers 😅 .